light_curves/ML_AGNzoo.md

```diff
@@ -50,7 +51,7 @@ This cell will install them if needed:
 
 ```{code-cell} ipython3
 # Uncomment the next line to install dependencies if needed.
-# !pip install -r requirements_ML_AGNzoo.txt
+#!pip install -r requirements_ML_AGNzoo.txt
```

Comment:
Adding a linter is on the list, but in the meantime I'll add a commit that cleans up pep8 things.

(cc @troyraen @zoghbi-a - do you know a jupyterlab linter extension that you would recommend? I think it's time to bite the bullet and set up something (I don't want to go nuclear and black all the notebooks, but we could enable to fix some non-controversial things like comments and long lines, etc.; but that whole work only makes sense if we provide something that helps to write the code that way from the beginning rather than ending up going back and forth)

Reply:
> do you know a jupyterlab linter extension that you would recommend?

I don't. I usually work with notebooks as python files and lint before converting to markdown.